pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -602,6 +645,312 @@ def test_dt64ser_sub_datetime_dtype(self):
         result = pd.to_timedelta(np.abs(ser - dt))
         assert result.dtype == 'timedelta64[ns]'
 
+    # -------------------------------------------------------------
+    # TODO: This next block of tests came from tests.series.test_operators,
+    # needs to be de-duplicated and parametrized over `box` classes
+
+    @pytest.mark.parametrize(
+        'box, assert_func',
+        [(Series, tm.assert_series_equal),
```

Comment:
note that we now have ``tm.assert_equal`` to solve cases like this (certainly can change later though)

Reply:
Yep, I'm looking forward to getting rid of this hack